docs/generated/devkit/nx_devkit.md

```diff
@@ -111,6 +111,7 @@ It only uses language primitives and immutable objects
 - [formatFiles](../../devkit/documents/nx_devkit#formatfiles)
 - [generateFiles](../../devkit/documents/nx_devkit#generatefiles)
 - [getDependentPackagesForProject](../../devkit/documents/nx_devkit#getdependentpackagesforproject)
+- [getExecutorInformation](../../devkit/documents/nx_devkit#getexecutorinformation)
```

Comment:
Let's keep this one private because no one uses it according to https://grep.app/search?q=readExecutor&case=true&filter[lang][0]=TypeScript

Reply:
removed. change to packages/nx/src/devkit-internals.ts